.github/workflows/static.yml

```diff
@@ -0,0 +1,49 @@
+name: OQS-BoringSSL (Static)
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+      - 'oqs_template/**'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'oqs_template/**'
+  schedule:
+    - cron: '2 7 18,28 * *'
```

Comment:
Why scheduled?

Reply:
to detect issues early, as updates to dependencies such as libunwind and Go can result in build failures